### PR TITLE
core: add new rook subcommand `debug <node|svc> ...`

### PR DESCRIPTION
**Pull request by randymtz**
Add subcmd to config "debug_$svc = 20/20" on either:
[$svc.id], or [$svc] level. [--unset] removes override.
debug node discovers & targets scheduled pods, whereas
debug svc configures "debug_$svc" at $svc.section [$svc].

Signed-off-by: Randy J. Martinez <randy@cephtips.com>

### CODE REVIEW

File: kubectl-rook-ceph.sh [hunk 1]
```diff
@@ -31,14 +31,20 @@ function print_usage() {
...
   echo " --context=<context_name>                   : the name of the Kubernetes context to be used"
-
   echo "COMMANDS"
   echo "  ceph <args>                               : call a 'ceph' CLI command with arbitrary args"
   echo "  rbd <args>                                : call a 'rbd' CLI command with arbitrary args"
   echo "  operator <subcommand>..."
   echo "    restart                                 : restart the Rook-Ceph operator"
   echo "    set <property> <value>                  : Set the property in the rook-ceph-operator-config configmap."
   echo "  mons                                      : output mon endpoints"
+  echo "  debug <subcommand>..."
```
[the remark this answers is not in the document]
Comment (randymtz) [reply kubectl-rook-ceph.sh:41]: Great points all around, I'll work on those updates.

> RE: node

We have identified the need to target `nodeName` in a few Day2/Day3 scenarios so far. 
example: 
  - DC sw misconfig/re-cable & bond slaves flapping
  - BZ/eng request for better understanding on why heartbeats are being missed
  - peering issues isolated to 1 specific node in AZ/rack from scale-out. 
  - T/S node contention w/CNV

**Comment by subhamkrai** [comment]
I restarted the ci to debug

File: kubectl-rook-ceph.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (subhamkrai) [reply kubectl-rook-ceph.sh:41]: `log` instead of `debug` sounds good.

**Comment by subhamkrai** [comment]
strange, when I ran the same command in ci it passed I'll open pr for the ci to debug more
```
kubectl rook_ceph rook version
rook: v1.9.0-alpha.0.448.g89fadabcc
go: go1.17.11
```

File: kubectl-rook-ceph.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (travisn) [reply kubectl-rook-ceph.sh:41]: @randymtz Thanks for the scenario details, if there is a need, then I could see a `[--node <node>]` option making sense.

**Comment by subhamkrai** [comment]
We are moving the project to golang, so it is not a valid change. Closing!

**Pull request closed without merging**